File: src/gestionMain/gestionCliente/GestionCliente.java
package gestionMain.gestionCliente;

import java.util.Scanner;
import gestionApp.Pedido.Cliente;

public class GestionCliente {

    public static void GestionarCliente() {
        Scanner scan = new Scanner(System.in);
        int opcion;

        do {
            System.out.println("\tMENU PARA CLIENTE"+"\n1. Ingrese sus datos para hacer su pedido: "+"\n2. Editar datos Ingresados: "+"\n3. Mostrar info del Cliente: ");
            opcion = scan.nextInt();
            switch (opcion) {
                case 1: 
                    RegistroCliente.registrarCliente();
                    break;
                case 2:  
                    EditarCliente.editarCliente();
                    break;
                case 3: 
                    MostrarCliente.mostrarClientes();
                    break;
                case 4: break;
                case 5: break;
                case 6: break;
            }

        }while (opcion != 4);
    }
}    

File: practica-2-grupo-2-equipo-4/GestionMain/gestionCliente/GestionCliente.py
from gestionCliente.RegistroCliente import RegistroCliente

class GestionCliente:
    
    opcion = 0
        
    def gestionarCliente(opcion):
        
        while opcion != 2:
                print("MENU CLIENTE: \n1. Ingresar datos para continuar con la orden  \n2. Realizar orden \n3. Factura y Salir: \n")
                try:
                    opcion = int(input())
                    if opcion == 1:
                        RegistroCliente.registrarCliente(opcion)
                        
                        break
                    if opcion == 2:

                        break
                    if opcion == 3:

                        break
                
                except :
                    print("Ingrese una opcion nuevamente: ")
                    break
